app/src/main/java/ch/epfl/polychef/fragments/OnlineMiniaturesFragment.java

```diff
@@ -139,14 +150,17 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
 
         searchView = getView().findViewById(R.id.searchBar);
         filters = getView().findViewById(R.id.filters);
-        ingredientsFilter = getView().findViewById(R.id.filter_ingre);
-        recipesFilter = getView().findViewById(R.id.filter_recipe);
-        usersFilter = getView().findViewById(R.id.filter_users);
-        rateFilter = getView().findViewById(R.id.filter_rate);
 
+        Button ingredientsFilter = getView().findViewById(R.id.filter_ingre);
         ingredientsFilter.setOnClickListener(setFilter(FILTER_INGREDIENT));
-        usersFilter.setOnClickListener(setFilter(FILTER_USER));
+
+        Button recipesFilter = getView().findViewById(R.id.filter_recipe);
         recipesFilter.setOnClickListener(setFilter(FILTER_RECIPE));
+
+        Button usersFilter = getView().findViewById(R.id.filter_users);
+        usersFilter.setOnClickListener(setFilter(FILTER_USER));
+
+        Button rateFilter = getView().findViewById(R.id.filter_rate);
```

Comment:
I'm guessing you forgot to replace with `requireView()` since you started using it everywhere else.

Reply:
Apparently, the view cannot be null here.